src/headless/HeadlessPluginLayerProxy.h

```diff
@@ -8,6 +8,6 @@ class HeadlessPluginLayerProxy
 public:
     void updateDisplay()
     {
-        std::cerr << "HeadlessPluginLayerProxy::updateDisplay" << std::endl;
+       // std::cerr << "HeadlessPluginLayerProxy::updateDisplay" << std::endl;
```

Comment:
You should make a decision to leave it or keep it.

Reply:
Yeah indeed! Sorry about that. It should go

Reply:
Nothing to be sorry about, just pinpointing what I see :)